### PR TITLE
Add mlperf benchmark for offline for mixtral

### PR DESCRIPTION
**Pull request by qihqi**
no server
use 3 buckets of different cache length (512, 1280, 3072)

### CODE REVIEW

File: jetstream_pt/offline_inference.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    for row in data:
+      log.debug(f"empty_slots {len(empty_slots)}")
+      while not empty_slots:
+        # If slots are all full, decode until there are free slots
+        # to insert
+        decode()
+      # do one insert
+      log.debug(f"prefill {row.id}")
+      slot = empty_slots.pop()
+      first_token = prefill(slot, row.tokens, row.true_length)
```
Comment (FanhaiLu1) [inline jetstream_pt/offline_inference.py:164]: Nit: rename it as prefill_insert since it do both prefill and insert?

File: run_interactive.py [hunk 2]
```diff
@@ -20,84 +20,75 @@
...
+  end = time.perf_counter()
+  print("init done in", end - start)
 
-    # pylint: disable-next=all
-    prefill_result, _ = engine.prefill(
-        params=params, padded_tokens=tokens, true_length=true_length
+  input_data = []
+  for i, prompt in enumerate(prompts):
+    tokens, true_length = tokenizer.encode(prompt)
+    input_data.append(
```
Comment (FanhaiLu1) [inline run_interactive.py:76]: For offline run, we know what are the input token length and we can sort (or rank) the data set based on the token length. 

If there are multiple prompt finish decode at same time, we can do multiple prefill before decode. The decode slot is empty if only one prefill per time.

**Review by FanhaiLu1 — APPROVED** [review]

**Comment by FanhaiLu1** [comment]
Can you fix the lint and unit test error?

**Comment by qihqi** [comment]
> Can you fix the lint and unit test error?

I'll do it when we merge it to main

**Pull request merged**